main.go

```diff
@@ -50,23 +58,22 @@ func main() {
 			disgord.EvtGuildMemberUpdate,
 			disgord.EvtGuildMemberRemove,
 		},
+		// ! Non-functional due to a current bug, will be fixed.
 		Presence: &disgord.UpdateStatusPayload{
 			Game: &disgord.Activity{
 				Name: "write " + prefix + "ping",
 			},
 		},
-		// I WANT DM EVENTS
-		// these are events sent directly to your bot through
-		// direct messaging.
-		// I recommend just not using these unless you have a specific use case.
-		// WARNING: you only specify intents when you want DM capabilities. 
-		//  For anything else, populate the RejectEvents setting above.
-		//Intents: []disgord.Intent{
-		//	disgord.IntentDirectMessageReactions,
-		//	disgord.IntentDirectMessageTyping,
-		//	disgord.IntentDirectMessages,
-		//},
+		DMIntents: disgord.IntentDirectMessages | disgord.IntentDirectMessageReactions | disgord.IntentDirectMessageTyping,
+
+		/*
+			! I DONT WANT DM EVENTS:
+			DMIntents allows your bot to recieve commands in DM, as well as Send messages in DMs.
+			Remove the above line if not necessary
```

Comment:
can you remove this comment block and state:
// comment out DMIntents if you do not want to handle direct messages

I try to avoid comment blocks /**/ to keep everything consistent. So I prefer multiple lines with // rather than /**/

Reply:
Updated in [5db1db0](https://github.com/endigma/disgord-starter/commit/34cc0e94049962b1f680bc08e904c01edd55efb8)